packages/config/config/devices/0x0431/ecodim.json

```
@@ -101,29 +110,29 @@
 			"description": "Enable or Disable external switch to pair network. Default value: 0.",
```

```suggestion
			"description": "Enable or Disable external switch to pair network.",
```
I prefer not having default values or value ranges in the description.